librz/type/parser/types_parser.c

```diff
@@ -1350,13 +1350,15 @@ int parse_type_abstract_declarator_node(CParserState *state, TSNode node, const
 				parser_error(state, "ERROR: Abstract pointer declarator AST should contain at least one node!\n");
 				node_malformed_error(state, node, text, "pointer declarator");
 				free(type);
+				(*tpair)->type = type = NULL;
```

Comment:
No need to set `type`, just `(*tpair)->type` is enough. Same for the second case.